### PR TITLE
[IMP-8]: Update Admin Panel Notification Element Design

### PR DESCRIPTION
**Pull request by hatemquliyev**
- add login notification to admin panel login page
- add alert notification components
- optimize codes
- add navigation to dashboard page

### CODE REVIEW

File: client/src/Admin/components/alert/Alert.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import '../../../utilities/styles/style';
+
+.alert-notification{
+    right: $right;
+    transition: .3s;
+    top: 0;
+    animation: fadeIn .2s ease-in-out;
+}
+
+.alert_container{
```
Comment (ilkinnezer1) [inline client/src/Admin/components/alert/Alert.scss:10]: if you use alert-container as a parent class then change name of it

File: client/src/Admin/components/alert/Alert.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ const Alert: FC<IPropsType> = ({icon, color, alertType, lastLoginTimeInfo}: IPropsType) => {
+    const alertSection = useRef<HTMLDivElement>(null)
+
+    const moveAlert = () => {
+        if (alertSection.current) {
+            alertSection.current.style.right = "0";
+        }
+    }
+
+    const closeAlert = () => {
```
Comment (ilkinnezer1) [inline client/src/Admin/components/alert/Alert.tsx:23]: you don't need to write 2 functions for one purpose

File: client/src/utilities/styles/_style.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    $mr-1: 15px !default;
+}
+// Margins
+.mr-1 {margin-right: 15px !important;}
+.ml-1 {margin-left: 15px !important;}
+
+// Height
+.hvh-100 {height: 100vh;}
+
+$right: 50px !default
```
Comment (ilkinnezer1) [inline client/src/utilities/styles/_style.scss:11]: remove `50px` for now :)

File: client/src/App.tsx [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
 import AdminArea from './Admin/AdminArea'
 import './App.css'
 
-function App() {
+export interface IQueryClientProvider {
+  children: React.ReactNode
+}
 
+function App() {
+  const queryClient = new QueryClient()
```
Comment (ilkinnezer1) [inline client/src/App.tsx:11]: do we need QueryClient now?

**Review by ilkinnezer1 — CHANGES_REQUESTED** [review]
please, consider all things and change it now

File: client/src/App.tsx (hunk 4 above)
Comment (hatemquliyev) [reply client/src/App.tsx:11]: No, we don't

File: client/src/Admin/auth/LoginPage.tsx [hunk 5]
```diff
@@ -1,34 +1,85 @@
...
+          password: password.trim(),
+        },
+        {
+            headers: {
+                "Content-Type":"multipart/form-data"
+            } 
+        });
+  
+        if (res.status === 200) {
+            const unFormattedDate = new Date(res.data.last_login_time)
```
[the remark this answers is not in the document]
Comment (hatemquliyev) [reply client/src/Admin/auth/LoginPage.tsx:40]: which ? status or date ?

File: client/src/App.tsx (hunk 4 above)
Comment (ilkinnezer1) [reply client/src/App.tsx:11]: Then please remove it

File: client/src/Admin/components/alert/Alert.scss (hunk 1 above)
Comment (hatemquliyev) [reply client/src/Admin/components/alert/Alert.scss:10]: we don't need it

**Review by ilkinnezer1 — APPROVED** [review]

**Pull request merged**